lib/ip-locator.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const Ipinfo = require('ipinfo')
+const RequestIp = require('request-ip')
+
+class IpLocator {
+  constructor (options) {
+    const { enabledByDefault = true } = options
+    this.enabledByDefault = enabledByDefault
```

Comment:
Please add an empty line here. Separating the class properties from the destructuring